js/package.json

```diff
@@ -6,19 +6,22 @@
   "repository": "git+https://github.com/reteps/dockerfmt/tree/main/js",
   "author": "Peter Stenger <pete@stenger.io>",
   "license": "MIT",
-  "main": "dist/node.js",
-  "types": "dist/node.d.ts",
   "exports": {
     ".": {
       "types": "./dist/node.d.ts",
```

Comment:
`types` here should be removed then.

(Actually, the user can never access `browser`'s types AKAIK. VSCode on web?

Reply:
Wow, first time to see this option.